packages/main/src/TabContainer.ts

```diff
@@ -438,18 +462,20 @@ class TabContainer extends UI5Element {
 
 		this._setItemsForStrip();
 
-		if (!this.shadowRoot.contains(document.activeElement)) {
+		if (!this.shadowRoot!.contains(document.activeElement)) {
 			const focusStart = this._getRootTab(this._selectedTab);
 			this._itemNavigation.setCurrentItem(focusStart);
 		}
 	}
 
 	onEnterDOM() {
-		ResizeHandler.register(this._getHeader(), this._handleResize);
+		// TO DO: handleResize is promise, it shouldn't be
```

Comment:
the correct form of a to-do comment is to begin it as "TODO". all other comments in the project are written that way

Reply:
Done